apps/src/labs/header/headerShareV2.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import LabRegistry from '../LabRegistry';
+import {getStore} from '@cdo/apps/redux';
+import ProjectWrappedShareDialog from './ProjectWrappedShareDialog';
+import {showShareDialog} from '@cdo/apps/code-studio/components/shareDialogRedux';
+
+/**
+ * Save, then show the share dialog for a project that uses the ProjectManager.
+ * @param {string} shareUrl - The URL of the project to share.
+ */
+export function shareProjectV2(shareUrl) {
```

Comment:
It would probably be good to have a good explanation of what V2 means at the top of this file.

I'm not sure about a better name than `shareProjectV2` unless we can work in some other keyword we're already using for this chain of functionality, like `shareProjectForProjectManager` or something?